pravega/pravega
# Issue 7424: Cherry-pick PR #7396 to r0.14

This PR is Upgrading the following dependencies in the pravega Dependency		
CVE fix Version
apacheCurator		5.6.0
azureSdk		12.25.1
jackson	2.16.0
qosLogBack	1.2.13

**Change log description**  
See https://github.com/pravega/pravega/pull/7396

**Purpose of the change**  
solves #7424 

**What the code does**  
See https://github.com/pravega/pravega/pull/7396

**How to verify it**  
See https://github.com/pravega/pravega/pull/7396
Successful system test run system-tests-1 5324
